changelog/improvement.7619.md

```diff
@@ -0,0 +1,4 @@
+Bump TensorFlow version to 2.6.
+
+Users can no longer set `TF_DETERMINISTIC_OPS=1` and use GPU(s) because a `tf.errors.UnimplementedError`
```

Comment:
nitpick: The impression I get after reading this is - I can neither set `TF_DETERMINISTIC_OPS` nor use GPU(s). How about?
```suggestion
Users can no longer set `TF_DETERMINISTIC_OPS=1` and use GPU(s) at once because a `tf.errors.UnimplementedError`
```

Reply:
You're totally right! I'm thinking "at once" might be a bit confusing as well? Maybe:

"Users can no longer set `TF_DETERMINISTIC_OPS=1` if they are using GPU(s) because a `tf.errors.UnimplementedError`.."

?